CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+The following sections document changes that have been released already:
```

Comment:
I already did this in solid-client-access-grants-js, but I've removed this text in favour of saying "These features have not yet been released: " followed by an empty list item.

https://github.com/inrupt/solid-client-access-grants-js/blob/main/CHANGELOG.md

Reply:
Fixed in https://github.com/inrupt/solid-client-authn-js/pull/1849/commits/2ea781889519a5ff35c690377818d4e77437f908